src/core_ocean/Registry.xml

```diff
@@ -394,6 +394,10 @@
 					description="If true, use the Community Vertical MIXing routines to compute vertical diffusivity and viscosity"
 					possible_values="True or False"
 		/>
+		<nml_option name="config_cvmix_use_kpp_buoyancyForcing_fix" type="logical" default_value=".false." units="NA"
```

Comment:
@vanroekel do you want this default true in E3SM? If so, I'll make it true here as well.